packages/genesys-spark-components/src/components/beta/gux-tooltip-base/gux-tooltip-base.tsx

```diff
@@ -83,6 +85,9 @@ export class GuxTooltipBase {
   @State()
   isShown: boolean = false;
 
+  @Event()
+  guxshown: EventEmitter<boolean>;
```

Comment:
using other components as a reference I think we should have two separate events for show and hide.
see `gux-accordion-section` - `guxopened`/`guxclosed` and `gux-dropdown` - `guxexpanded`/`guxcollapsed`